start.sh

```diff
@@ -27,6 +27,7 @@ if [ -z "${IS_CRON_POD}" ]; then
         exec gunicorn student_explorer.wsgi:application \
             --bind 0.0.0.0:${GUNICORN_PORT} \
             --workers="${GUNICORN_WORKERS}" \
+            --timeout="${GUNICORN_TIMEOUT}" \
```

Comment:
I noticed Myla don't have this timeout too, what is the purpose of this timeout?

Reply:
Student Explorer has a task of uploading cohort members. And depending on the length of the cohort list, the job may need long time to finish. This is to set a longer timeout, but the ideal fix should be improving the upload process performance.

Reply:
This is to change the default timeout of gunicorn from 30 seconds to some larger number. There's a process in Student Explorer for loading data files. It's currently inefficient and rather than spend the time to improve it we've just increased this timeout so the client isn't disconnected. Nothing in MyLA should take longer than 30 seconds to respond, so we haven't set it.

Reply:
MyLA uses cron to load data. There is no user requested long-running job now in MyLA

Reply:
We had ideas here like #14 to fix this as well as some investigation into how the process actually works now (lots of individual inserts and selects rather than fewer bulk queries). But since very few cohorts are large enough to hit this timeout, and the ones that are are loaded only a few times a year by admins, the tradeoff has just persisted here.